CHANGELOG.md

```diff
@@ -15,9 +15,15 @@
   [\#565](https://github.com/CosmWasm/wasmd/pull/565)
 
 **Implemented Enhancements:**
-
+- Wasmvm upgrade 557 [\#559](https://github.com/CosmWasm/wasmd/pull/559) ([ethanfrey](https://github.com/ethanfrey))
```

Comment:
Where does "557" come from?

Reply:
Good 👁️ . This was part of the PR title and got copied by the tool. 🤷

Reply:
It was the issue number I used